sonar-update-center-common/src/main/java/org/sonar/updatecenter/common/UpdateCenterDeserializer.java

```diff
@@ -183,6 +185,18 @@ private void validatePublicPluginSQVersionOverlap(Plugin plugin) {
     }
   }
 
+  private void validateLATESTonLatestPluginVersion(List<Plugin> plugins) {
+    for (Plugin plugin : plugins) {
+      for (Release r : plugin.getReleases()) {
+        Version [] versionsWLatest = r.getSonarVersionFromString("LATEST");
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "LATEST" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1192)